invenio_app_rdm/administration/records/records.py

```diff
@@ -91,3 +91,60 @@ def init_search_config(self):
             page=1,
             size=30,
         )
+
+
+class DraftAdminListView(AdminResourceListView):
+    """Configuration for OAI-PMH sets list view."""
+
+    api_endpoint = "/user/records"
+    name = "Drafts"
+    resource_config = "records_resource"
+    search_request_headers = {"Accept": "application/json"}
```

Comment:
```suggestion
    search_request_headers = {"Accept": "application/vnd.inveniordm.v1+json"}
```
in zenodo we override the `application/json` serializer returning the legacy, see https://github.com/zenodo/zenodo-rdm/blob/master/site/zenodo_rdm/legacy/resources.py#L51